specs/Deployment/DeploymentAPI.md

```diff
@@ -155,6 +203,16 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         HRESULT ExtendedError{ get; };
     };
 
+    // TODO: https://task.ms/38272182 - add APIcontract once WinAppSDK's rules for them are defined [contract(name,version)]
+    /// This object is used to specify deployment options to apply when using DeploymentManager's
+    /// Initialize method
+    runtimeClass DeploymentInitializeOptions
+    {
+        /// Gets or sets a value that indicates whether the processes associated with the package will be
```

Comment:
The comment says 'the package", but there is no package parameter to DeploymentManager.Initialize. This is for installing the WindowsAppRuntime itself.

```cpp
// If true, installation of the WindowsAppRuntime will shut down any
// processes that are using it.
```

Reply:
RECOMMEND: Refer to the runtime, not a package (as the runtime encompasses multiple packages)

CCONSIDER: Revise the spec to match